airflow-core/src/airflow/ui/public/i18n/locales/en/dags.json

```
@@ -62,6 +62,7 @@
       "onlyFailed": "Clear only failed tasks",
       "past": "Past",
       "queueNew": "Queue up new tasks",
+      "runOnLatestVersion": "Run with latest version",
```

```suggestion
      "runOnLatestVersion": "Run with latest bundle version",
```

Probably worth being more specific that this is the bundle version?